src/cdk/drag-drop/directives/drop-list.ts

```diff
@@ -123,6 +124,15 @@ export class CdkDropList<T = any> implements AfterContentInit, OnDestroy {
   @Input('cdkDropListAutoScrollDisabled')
   autoScrollDisabled: boolean = false;
 
+  /**
+   * CSS selector that will be used to determine which parent elements can be scrolled while the
+   * user is dragging an item in the list. The viewport and the list itself are always allowed
+   * to be scrollable, but for performance reasons scrolling won't be monitored on any other
+   * parent elements that don't match this selector.
+   */
+  @Input('cdkDropListScrollableParents')
+  scrollableParents: string;
```

Comment:
I'm not too sure what the API should look like here. Initially I started off with making it a directive that you'd put on an element that should be scrollable and then the CDK would use DI to find it. The problem with the directive approach was that it won't go past view boundaries so I went with this approach instead. I'm open to suggestions on how we can make it better.

Reply:
why not use cdk-scrollable https://material.angular.io/cdk/scrolling/api#CdkScrollable as parents?

Reply:
I was considering that, but it would mean people having to import another module and what I've  seen over time from the various GitHub issues is that people don't really know how to use it. I don't know whether it's a documentation issue or not getting any feedback if they add the attribute without the module being included.